src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -4351,6 +4357,29 @@ emit_arm64_intrinsics (
 			MONO_ADD_INS (cfg->cbb, ins);
 			return ins;
 		}
+		case SN_StoreVector128x2:
+		case SN_StoreVector128x3:
+		case SN_StoreVector128x4:
+		case SN_StoreVector64x2:
+		case SN_StoreVector64x3:
+		case SN_StoreVector64x4: {
+			int opcode = 0;
+			switch (id) {
+			case SN_StoreVector128x2: opcode = INTRINS_AARCH64_ADV_SIMD_ST1X2_V128; break;
+			case SN_StoreVector128x3: opcode = INTRINS_AARCH64_ADV_SIMD_ST1X3_V128; break;
+			case SN_StoreVector128x4: opcode = INTRINS_AARCH64_ADV_SIMD_ST1X4_V128; break;
+			case SN_StoreVector64x2: opcode = INTRINS_AARCH64_ADV_SIMD_ST1X2_V64; break;
+			case SN_StoreVector64x3: opcode = INTRINS_AARCH64_ADV_SIMD_ST1X3_V64; break;
+			case SN_StoreVector64x4: opcode = INTRINS_AARCH64_ADV_SIMD_ST1X4_V64; break;
+			default: g_assert_not_reached ();
+			}
+
+			MonoClass* value = mono_class_from_mono_type_internal (fsig->params [1]);
+
+			MonoInst* ins = emit_simd_ins_for_sig (cfg, klass, OP_ARM64_STM, opcode, arg0_type, fsig, args);
```

Comment:
You could pass `value` as the second argument for `emit_simd_ins_for_sig` and get rid of the next line.